larpandoracontent/LArVertex/CandidateVertexCreationAlgorithm.cc

```diff
@@ -109,12 +112,37 @@ void CandidateVertexCreationAlgorithm::SelectClusters(ClusterVector &clusterVect
         ClusterVector sortedClusters(pClusterList->begin(), pClusterList->end());
         std::sort(sortedClusters.begin(), sortedClusters.end(), LArClusterHelper::SortByNHits);
 
+        int count1(0);
```

Comment:
Is there a more self-describing name than `count1`? What is this quantity? `nClustersPassingBasicCuts`?